MAUI/Autocomplete/Selection.md

```diff
@@ -0,0 +1,222 @@
+---
+layout: post
+title: Selection in .NET MAUI Autocomplete control | Syncfusion
+description: Learn here all about Selection support in Syncfusion .NET MAUI Autocomplete (SfAutocomplete) control and more.
```

Comment:
description: Learn all about Selection support in Syncfusion .NET MAUI Autocomplete (SfAutocomplete) control and more here.